README.md

```diff
@@ -89,6 +89,31 @@ This will then prompt you to enter a Trello API Token, Trello API Key, and Trell
 
 When the playbook has completed, you will then be able to access the app at port 5000 on the IP address of your managed node.
 
+## Running the App in a Docker container
+
+The app can be run in both a production and a development docker container.
+
+To run the app in development mode you must build the development image and then spin up a container from it as follows:
+
+```bash
+docker build --target development --tag todo-app:dev .
+docker run --env-file ./.env -p 5000:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/opt/todo_app" --detach todo-app:dev
```

Comment:
I see errors when I run this, and I think it's because of a mismatch of folders:
* In your Dockerfile, your working directory is `/opt/todo_app`
  * This means that e.g. your poetry files live at `/opt/todo_app/poetry.lock` etc.
  * And that your source code is nested one further and lives at `/opt/todo_app/todo_app`
* But this overrides your poetry files with your source code!
```suggestion
docker run --env-file ./.env -p 5000:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/opt/todo_app/todo_app" --detach todo-app:dev
```